docs/UserGuide.adoc

```diff
@@ -130,17 +243,238 @@ Examples:
+****
+
+Example:
+
+`add n/John Doe ic/Q9999999J p/98765432 e/@example.com a/John street, block 123, #01-01 b/12.09.1980 pol/SeniorCare t/high-priority` +
+
+Expected Output:
+```
+@example.com is invalid. You might want to contact John Doe: 98765432
+```
+
+=== Incorrect Email Address Warning
+
+Returns a warning message to inform the user of a potential typo in email address.
+
+****
+* The details of the input person will not be added until all information is confirmed.
+* These warnings can be turned off.
+****
+
+Example:
+
+`add n/John Doe ic/Q9999999J p/48765432 e/johnd@gmal.com a/John street, block 123, #01-01 b/12.09.1980 pol/SeniorCare t/high-priority` +
+
+Expected Output:
+```
+You entered johnd@gmal.com. Did you mean: johnd@gmail.com?
+```
+
+User can input `yes` or `no` to use the original input email or the suggested email respectively.
+
+=== Incorrect Command Suggestions
+
+Returns an error message and a suggestion of a correct command when a command is typed incorrectly.
+
+Example:
+
+`dlete 2`
+
+Expected Output:
+```
+dlete is not recognised. Did you mean: delete 2?
+```
+
+User can input `yes` or `no`.
+
+If `yes`, the command will be executed.
+If `no`, `Please key in your command again. For more information  of the available commands, type help.`
+
+=== Incorrect Insurance Policy Assignment Alert
+
+Returns an error if a person is ineligible for the insurance policy and provides suggestions for policies that this person is eligible for.
+
+****
+* The new information will be accepted and updated except for the updated policy.
+****
+
+Example:
+
+`edit n/John Doe ic/S9999999J p/91234567 e/johndoe@example.com a/John street, block 123, #01-01 age/30 pol/SeniorCare`
+
+Expected Output:
+```
+John Doe is ineligible for this policy. These are other possible policies that John Doe is eligible for:
+...
+```
+
+=== Duplicate profile alert
+
+Returns an error message of an existing person and will attempt to merge the profiles.
+For each different attribute, there will be a prompt to suggest a change from the original attribute to the new one.
+
+Example:
+
+`add n/John Doe ic/S9999999J p/91234567 e/johndoe@example.com a/John street, block 123, #01-01 age/30 pol/SeniorCare`
+
+Expected Output:
+```
+This profile already exists:
+n/John Doe ic/S9999999J p/98765432 e/johndoe@example.com a/John street, block 123, #01-01 age/30`
+Do you wish to update John Doe’s profile?
+```
+
+User may input `yes` or `no`. Further prompts will be provided if user inputs  `yes`
+
+```
+Would you like to update p/98765432 to p/91234567?
+```
+
+If `yes`, the new profile will be created.
+If `no`, `You may edit John Doe’s profile using the edit command. For more information, type help.`
+
+=== Reminders of expiring policies
+
+Displays a list of people whose policies are expiring in the current month on the start up page. This list will be updated every month.
+
 === Exiting the program : `exit`
 
 Exits the program. +
```

Comment:
Paiseh ah, but do u mind helping to add ur the basic commands and ur commands to the command summary?